egs/chime4/s5_2ch/path.sh

```diff
@@ -1,6 +1,8 @@
 export KALDI_ROOT=`pwd`/../../..
+[ -f $KALDI_ROOT/tools/extras/env.sh ] && . $KALDI_ROOT/tools/extras/env.sh
 [ -f $KALDI_ROOT/tools/env.sh ] && . $KALDI_ROOT/tools/env.sh
 export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PWD:$PATH
 [ ! -f $KALDI_ROOT/tools/config/common_path.sh ] && echo >&2 "The standard file $KALDI_ROOT/tools/config/common_path.sh is not present -> Exit!" && exit 1
 . $KALDI_ROOT/tools/config/common_path.sh
 export LC_ALL=C
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/dpovey/libs/
```

Comment:
this is a local thing that doesn't belong here.

Reply:
oh! Sorry. I do not know about this.